skimage/feature/texture.py

```diff
@@ -291,3 +295,154 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+
+def multiblock_lbp(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    The features are calculated similarly to local binary patterns (LBPs),
```

Comment:
Since we reference LBPs, either/both of a See Also section in the docstring pointing toward LBP would be appropriate, as well as another reference for LBP. I'd be happy with the former; consider both extra credit...